build-system/tasks/compile-config.js

```diff
@@ -32,5 +34,7 @@ exports.resolveConfig = function() {
     'frontend': argv.frontend || FRONTEND,
     'frontendDebug': argv.frontendDebug || FRONTEND_DEBUG,
     'entitlements': argv.entitlements || ENTITLEMENTS,
+    'payEnvironment': argv.payEnvironment || PAY_ENVIRONMENT,
```

Comment:
where does argv get set?

Reply:
It's a command line. Makes testing easier.